rr/templates/streamfields/html/image_lazy.html

```diff
@@ -7,20 +7,29 @@
 {% image self.image scale-30 format-webp as image_webp_30 %}
 {% image self.image scale-30 as image_30 %}
 
+{% if self.page_url or self.external_url %}
+<a href="{% if self.external_url %}{{ self.external_url }}{% else %}{{ self.page_url.url }}{% endif %}"{% if self.open_in_new_tab %} target="_blank"{% endif %}>
+{% endif %}
+
 {% if self.lazy %}
```

Comment:
The lazy logic is totally wrong

Reply:
Is this block even supposed to be in here?
Because the frontend would need jquery-lazy to work.